shared/teams/team/settings-tab/channel-popup.tsx

```diff
@@ -0,0 +1,100 @@
+  const [filter, setFilter] = React.useState('')
+  const filterLCase = filter.toLowerCase()
+  const onChangeFilter = (value: string) => setFilter(value)
+
+  const channelInfos = Container.useSelector(s => Constants.getTeamChannelInfos(s, teamID))
+  const channels = [...channelInfos.values()].map(ci => ci.channelname)
+  const channelsFiltered = filter ? channels.filter(c => c.toLowerCase().includes(filterLCase)) : channels
+
+  const [selected, setSelected] = React.useState(new Set<string>())
+  const onSelect = (channelname: string) => {
+    if (selected.has(channelname)) {
+      selected.delete(channelname)
+    } else {
+      selected.add(channelname)
+    }
+    setSelected(new Set(selected))
+  }
+
+  const onAdd = () => onComplete([...selected])
+  return (
+    <Kb.MobilePopup overlayStyle={Styles.globalStyles.fullHeight}>
+      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.header} gap="tiny">
+        <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.headerTop}>
+          <Kb.Text type="BodyBigLink" onClick={onCancel}>
+            Cancel
+          </Kb.Text>
+          <Kb.Text
+            type="BodyBigLink"
+            onClick={selected.size ? onAdd : undefined}
+            style={!selected.size && styles.addDisabled}
+          >
+            Add{!!selected.size && ` (${selected.size})`}
+          </Kb.Text>
+        </Kb.Box2>
+        <Kb.SearchFilter
+          placeholderText={`Search ${channels.length} ${pluralize('channel', channels.length)}`}
+          size="full-width"
+          onChange={onChangeFilter}
+          style={styles.searchFilter}
+          placeholderCentered={true}
+          icon="iconfont-search"
+        />
+      </Kb.Box2>
+      <Kb.BoxGrow>
+        <Kb.ScrollView style={Styles.globalStyles.fillAbsolute}>
+          {channelsFiltered.map(channelname => {
+            const disabled = disabledChannels?.includes(channelname)
+            const onClick = disabled ? undefined : () => onSelect(channelname)
+            return (
+              <Kb.ClickableBox key={channelname} onClick={onClick}>
+                <Kb.Box2 direction="horizontal" style={styles.channelContainer} gap="tiny" fullWidth={true}>
+                  <Kb.Text type="Body" lineClamp={1} style={Styles.globalStyles.flexOne}>
+                    #{channelname}
+                  </Kb.Text>
+                  <Kb.CheckCircle
+                    onCheck={onClick}
+                    checked={disabled || selected.has(channelname)}
+                    disabled={disabledChannels?.includes(channelname)}
+                  />
+                </Kb.Box2>
+              </Kb.ClickableBox>
+            )
+          })}
+        </Kb.ScrollView>
+      </Kb.BoxGrow>
+    </Kb.MobilePopup>
+  )
+}
+
+const styles = Styles.styleSheetCreate(() => ({
+  addDisabled: {opacity: 0.4},
+  channelContainer: {
+    ...Styles.padding(14, Styles.globalMargins.medium, 14, Styles.globalMargins.small),
```

Comment:
I checked the design and I agree this is 14.. What ever happened to that world where the margins made sense 😢